web-console/src/views/lookups-view.tsx

```diff
@@ -38,6 +38,8 @@ import './lookups-view.scss';
 
 const tableColumns: string[] = ['Lookup name', 'Tier', 'Type', 'Version', 'Actions'];
 
+const default_lookup_tier: string = '__default';
```

Comment:
there should hopefully be a tslint rule to this effect but all variables and constants should be either UPPER_CASE_WITH_UNDERSCORES or snakeCase in this case since this is a 'true' constant it should be the former.